apps/docs/src/app/core/component-docs/dialog/examples/component-based/dialog-example.component.ts

```diff
@@ -9,7 +9,7 @@ import { DialogRef } from '@fundamental-ngx/core/dialog';
             </fd-dialog-header>
 
             <fd-dialog-body>
-                <p id="fd-dialog-body-1" role="dialog" style="text-align: justify; margin: 0">
+                <p id="fd-dialog-body-1" role="document" style="text-align: justify; margin: 0">
```

Comment:
cannot add if we add it from the component it just reads as "contentInfoRegion" need to specifically specify role as "document"  if its a static content.

Reply:
i did not really understand why we are using `role="document"`. Which issue is this addressing?
From the [a11y ref link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Document_Role#keyboard_interactions), it says `role="document"` should be used with `tabindex="0"` in order to read the static content. I don't see `tabindex` here, nor do I understand why we need `role="document"` in the first place: 
> Generally placed within an application role or other interactive widget role, the document role is used to indicate a section of a complex composite widget that an assistive technology user should read using its browse or virtual reading mode, if available.
Because ATs with reading mode default to that mode for all elements except for those with a widget or application role set, **document role is only useful for focusable elements within a widget or application that should be read as static rich text**.

Reply:
you haven't addressed this. Please address or respond to all comments before asking for re-review.

Reply:
this cannot be removed, on checking with Accessability team they raised the issue to read the content as details, details in the sense the static content inside the dialog body.